test/database_service/test_db_experiment_data.py

```diff
@@ -1058,12 +1060,25 @@ def test_additional_attr(self):
     def test_copy_metadata(self):
         """Test copy metadata."""
         exp_data = FakeExperiment(experiment_type="qiskit_test").run(backend=FakeBackend())
+        self.assertExperimentDone(exp_data)
         exp_data.add_data(self._get_job_result(1))
         copied = exp_data.copy(copy_results=False)
         self.assertEqual(exp_data.data(), copied.data())
         self.assertFalse(copied.analysis_results())
         self.assertEqual(exp_data.provider, copied.provider)
 
+    def test_copy_analysis_results(self):
```

Comment:
This is good. Should there be a test of copy in `test_analysis_results_table.py` as well?